### PR TITLE
Added new student to the page

### PR DESCRIPTION
**Pull request by Shizerq**
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.

# Changes done in this Pull Request

I've added myself to the page as a student:
-Added lines to student.yml
` name: Maciej Synowski
  github: Shizerq
  image: Shizerq.jpg
  twitter: maciej_synowski`
-Added image "Shizerq.jpg"

Preview: https://shizerq.github.io/gci18.fossasia.org/

Fixes issue: none. 
That's task from Google Code-In

### CODE REVIEW

**Comment by Shizerq** [comment]
Updated description (does adding 3 lines of code and a photo really needs long description?). Deleted .DS_store files.

File: _data/students.yml [hunk 1]
```diff
@@ -53,7 +53,14 @@
...
+
 - name: Filip Michalak
   github: Rekjef
   image: RekjefAvatar.png
+
+- name: Maciej Synowski
+  github: Shizerq
+  image: Shizerq.jpg
+  twitter: maciej_synowski
+=======
```
Comment (jogendra) [inline _data/students.yml]: Why?

File: _data/students.yml [hunk 2]
```diff
@@ -53,7 +53,14 @@
...
+  >>>>>>> master
```
Comment (jogendra) [inline _data/students.yml]: remove it too.

Comment (Shizerq) [reply _data/students.yml]: It stayed accidentally, while i was fixing conflict. Deleted.

File: _data/students.yml [hunk 3]
```diff
@@ -53,7 +53,13 @@
...
+
 - name: Filip Michalak
   github: Rekjef
   image: RekjefAvatar.png
+
+- name: Maciej Synowski
+  github: Shizerq
+  image: Shizerq.jpg
+  twitter: maciej_synowski
+
```
Comment (jogendra) [inline _data/students.yml]: remove the empty line.

Comment (Shizerq) [reply _data/students.yml]: Done.

**Pull request merged**